resource-aggregate/.plantuml

```diff
@@ -296,4 +296,28 @@ ResourceAggregate --> Bus : DeviceMetadataUpdated (Canceled=true)
 ResourceAggregate -> Client : CancelPendingMetadataUpdatesResponse
 deactivate Client
 deactivate ResourceAggregate
-@enduml
\ No newline at end of file
+@enduml
+
+@startuml
+title "Delete Devices"
+
+entity "OCF Server" as Server
+participant "Gateway" as Gateway
+participant "Resource Aggregate" as ResourceAggregate
+participant "Authorization Server" as AuthServer
+
+activate ResourceAggregate
+activate Server
```

Comment:
activation represents processing of the request. You have to do it step by step. :)